guides/doc-Upgrading_and_Updating/topics/upgrading_a_disconnected_satellite.adoc

```diff
@@ -22,7 +22,7 @@ endif::[]
 .Before You Begin
 
 * Review and update your firewall configuration before upgrading your {ProjectServer}.
-For more information, see https://access.redhat.com/documentation/en-us/red_hat_satellite/{AccessRedHatComVersion}/html-single/installing_satellite_server_from_a_disconnected_network/index#satellite-ports-and-firewalls-requirements_satellite[Ports and Firewalls Requirements] in _Installing {ProjectServer} from a Disconnected Network_.
+For more information, see {BaseURL}installing_satellite_server_from_a_disconnected_network/index#satellite-ports-and-firewalls-requirements_satellite[Ports and Firewalls Requirements] in _Installing {ProjectServer} in a Disconnected Network Environment_ guide.
```

Comment:
Apologies for the partial review earlier, but we even have `InstallingDisconnectedDocURL` which is even more precise.
```suggestion
For more information, see {InstallingDisconnectedDocURL}satellite-ports-and-firewalls-requirements_satellite[Ports and Firewalls Requirements] in _Installing {ProjectServer} in a Disconnected Network Environment_ guide.
```

In https://github.com/theforeman/foreman-documentation/pull/856 I intended to clean it up, but never got the traction to get merged. I still think it's a good idea so I rebased that. I don't mind rebasing later if this is merged, but also feel free to pull in the first commit of that.

Reply:
No worries, I will get right on it.